### ISSUE TITLE
Air Printing - despite calibration

### ISSUE THREAD

**Issue by WanaGo** [BCN3D/BCN3DSigma-Firmware#147]
Have you seen this problem before, where the printer will air print, despite doing a full calibration, bed calibration and G36 calibration.... it still just prints in the air...

During the calibration, it prints perfectly on the bed. But during a print, it just goes and prints in the air...

Is the board suffering from EEPROM wear or something, given the application writes to EEPROM so often?

**Comment by lampi87** [comment]
Same for me
I've reverted version to 1.2.3 and changed generated profile (progen) to an old one. Now it seems to work fine.

**Comment by AlejandroGarcia92** [comment]
Hi, we don't have these issues. I don't know if this issue can be produced by profile, firmware or mechanical causes. I remember that EEPROM can be write/erase up to 100000 cycles.

**Comment by WanaGo** [comment]
Hi Alejandro,

Sorry I forgot to update this issue.
The issue ended up being Simplify3D, when the Z-Hop / Retraction Vertical Lift is set to 0, and you have a Skirt, it prints the skirt in the air, which is what I was seeing at the start of the print. I have already reported this to S3D, however by the sounds of it, it will never be fixed.
![image](https://user-images.githubusercontent.com/759846/30314033-892ba342-97f3-11e7-9a22-35fa56e4310e.png)

When you do dual colour, and have the same setting on 0, it goes even more crazy.
![image](https://user-images.githubusercontent.com/759846/30314063-a0d7d880-97f3-11e7-9365-3b75ad835550.png)

This was resolved by setting the distance to 0.01mm instead of 0mm.

Other than these, I have not seen printing in the air on 1.2.5, however I have heard of others who have, and mentioned it takes a number of bed level calibrations to get the bed in the right place again. However, this was not my issue.
